engine.json

```diff
@@ -87,6 +87,7 @@
         "Gems/WhiteBox"
     ],
     "projects": [
+        "Test7312",
```

Comment:
This change should not be included

Reply:
@jjjoness You can transition this PR to [draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft) state if it is not ready to be reviewed:

Reply:
@AMZN-alexpete All ready for review.